File: internal/languages/php/detectors/testdata/string.php
<?php
class Greet {
    const Greeting = "Hello World";

    public static function main($args)
    {
        $s = self::Greeting . "!";
        $s .= "!!";

        $s2 = "hey ";
        $s2 .= $args[0];
        $s2 .= " there";
    }
}
?>

File: internal/languages/golang/detectors/testdata/string.go
package main

import (
	"fmt"
	"os"
)

var Greeting = "Hello World"

func main() {
	s := Greeting + "!"
	s += "!!"
	fmt.Println(s)

	s2 := "hey "
	s2 += os.Args[0]
	s2 += " there"
	fmt.Println(s2)

	s3 := "foo " + os.Args[0] + " bar"
	fmt.Println(s3)
}
